README.md

```diff
@@ -40,17 +49,23 @@ As part of the code the system produces a JSON file which then is used to carry
 | TYPE | If it is a Full or Light Hotspot |
 
 
-
 ## Local development environment
 
 Because the stack is tightly intertwined with Balena, the easiest way to test the code base on your own Raspberry Pi in your own Balena project.
 
-* Create a new Balena project for Raspberry Pi 3 (64 Bit)
-* Download and flash out the disk image provided and boot the device
-* Add the remote Balena repo (`git remote add balena YourUser@git.balena-cloud.com:YourUser/YourProject.git`)
+* Create a new Balena application (in a personal org):
+    * Default device type: `Raspberry Pi 3 (using 64 bit OS)`
+    * Application type: `Starter`
+* Add a device:
+    * Select newest version
+    * Production
+    * Click `Download Balena OS`
+* Use [Etcher](https://www.balena.io/etcher/) to flash the downloaded image.
+* Insert flash drive into the Raspberry Pi and boot (don't forget to plugin ethernet if necessary).
+* Deploy changes: `balena push BALENA_PROJECT_OR_DEVICE`
 
-You can now push your changes using the following command:
+If you are on the same network as the Raspberry Pi, enter `LOCAL IP ADDRESS` from Balena into the browser.
```

Comment:
Does anybody have instructions for connecting remotely handy?

Reply:
We probably don't want to connect remotely, it's designed to be a local debug / status tool, for remote operation see https://dashboard.nebra.com

Reply:
Our support team never has to do this @vpetersson? It would be nice to be able to leave something plugged in at my coworking space and debug from home. I get that we don't want them to be publicly accessible by default but I'm guessing there's a way to connect remotely if you have Balena access.

Reply:
It's possible through balena yes. Using their PUBLIC DEVICE URL toggle

https://www.balena.io/docs/learn/manage/actions/#enable-public-device-url

Reply:
Also, one thing to note is you may need some environment variables to not throw errors. I'm not 100% sure but the to get the frequency I think it needs FREQ set to 868 or whatever it is. And the VARIANT set to whatever hardware it is from the hardware definitions repo

Reply:
Thanks for pointing that out. [These](https://github.com/NebraLtd/hm-diag/blob/5436da7e8a06cae35791cf00381abe77ed4885f2/diagnostics-program/main.py#L52) are all the env vars that the app references. I'll double check which ones are automatically being set.

```
[
        'BALENA_DEVICE_NAME_AT_INIT',
        'BALENA_DEVICE_UUID',
        'BALENA_APP_NAME',
        'FREQ',
        'FIRMWARE_VERSION',
        'VARIANT'
    ]
```

Reply:
I think freq and variant come from the balena app environment variable settings.

The firmware version is from the docker compose.

Guess the others are automatically pulled in from balena

Reply:
@marvinmarnold all the ones with the `BALENA_*` prefix should be set automatically from Balena. The rest I believe we set either on a per-device level (such as in TestNet) or in the App (the production deployments).